app/build.gradle

```diff
@@ -17,12 +17,16 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile project(":lib")
-    compile 'com.android.support:appcompat-v7:26.0.1'
+    compile fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
If we are using Gradle 4 then we should also replace it with "implementation", but it is not your mistake - it was already this way